examples/remix-minimal-starter/app/routes/index.tsx

```diff
@@ -1,16 +1,20 @@
 export default function Index() {
   return (
     <div style={{ fontFamily: 'system-ui, sans-serif', lineHeight: '1.4' }}>
-      <h1>Welcome to Remix</h1>
+      <h1>Welcome to Remix + Builder.io Starter</h1>
+      <h4>
+        This is index static route for remix pages, to visit your Builder.io pages, publish some and
+        have fun!
```

Comment:
```suggestion
        This is the index static route for Remix pages. To see your Builder.io content, make sure to create and publish a page at builder.io/content.
```
I am not entirely sure how this page works, but it feels like the text should read a little more like what I have here. Does that make sense?